addon/mixin.js

```diff
@@ -16,9 +16,11 @@ export default Ember.Mixin.create({
     });
   },
 
+  keyPropertyName: 'content',
```

Comment:
would prefer for this to be on the factory, that way it cannot collide with instance state.

```js
Service.reopenClass({
  keyPropertyName: '..'
})
```